src/psyGen.py

```diff
@@ -1864,16 +2101,31 @@ def iteration_space_arg(self, mapping={}):
                               "we assume there is at least one writer, "
                               "reader/writer, or increment as an argument")
 
-    def set_dependencies(self):
-        for argument in self._args:
-            argument.set_dependencies()
-        # TODO create a summary of dependencies
+    def args_filter(self, arg_types=None, arg_accesses=None):
```

Comment:
Whilst I agree that this (and presumably args_filter in the Loop class) should be replaced by psyGen.args_filter, this appears to still be used on my branch by args_filter in the loop class which is then used in dynamo0p3.py and transformations.py so I don't think it is simply a case of removing it. args_filter already existed so I'm not sure why it is showing up as a change in this issue.